backend/lib/endpoints/users.js

```diff
@@ -35,6 +38,17 @@ async function routes(app) {
       };
     },
   );
+
+  app.post(
+    "/:userId",
```

Comment:
`userId` should not be in the path because the `userId` is geenrated in Auth0 as `mongo_id` and encoded inside the JWT
so I think this should just be the `/api/users/` route